Builds/CMake/CMakeFuncs.cmake

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
-  message(WARNING "In-source builds are strongly discouraged.")
+  message(WARNING "In-root builds are strongly discouraged.")
```

Comment:
I've always heard these referred to as "in-source" not "in-root". Here's an example from their FAQ: https://cmake.org/Wiki/CMake_FAQ#I_run_an_out-of-source_build_but_CMake_generates_in-source_anyway._Why.3F

It's fine as-is, more of an FYI.